src/main/resources/org/rust/ide/liveTemplates/Rust.xml

```diff
@@ -60,4 +60,28 @@
         </context>
     </template>
 
+    <template name="loop" description="Infinite loop" toReformat="true" toShortenFQNames="true"
+              value="loop {&#10;  $SELECTION$&#10;}$END$">
```

Comment:
Yeah, but if the user wants to modify the template then four spaces will be less surprising. 

Also our formatter is far from good yet, so sometimes it is more robust to use `toReformat="false"`